src/index.tsx

```diff
@@ -50,15 +50,24 @@ export class PDFObject extends React.PureComponent<Props> {
     assumptionMode: true,
   };
 
-  public componentDidMount() {
+  embed = () => {
```

Comment:
please add `private` access modifier

Reply:
@sugarshin  no problem, I've done for changed 💯